ansible/roles_ocp_workloads/ocp4_workload_synopsys_blackduck/tasks/workload.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Setting up Synopsys Black Duck workload for user
+  debug:
+    msg: "Setting up Synopsys Black Duck workload for user ocp_username = {{ ocp_username }}"
+
+# Synopsys Black Duck workload tasks
+# -------------------------------------------------------------------------
+
+- name: Install synopsysctl CLI
+  become: true
+  unarchive:
+    src: "{{ ocp4_workload_synopsys_blackduck_ctl_url }}"
+    remote_src: yes
+    dest: /usr/bin
+    mode: 0775
+    owner: root
+    group: root
+  args:
+    creates: /usr/bin/synopsysctl
+
+- set_fact:
+    ocp4_workload_current_project: "{{ ocp4_workload_synopsys_blackduck_demo_project }}"
```

Comment:
1. Ah, thanks @wkulhanek !! I'll add names asap.
2. Since I'm creating 2 namespaces, I wanted to use the same project.j2 template. Happy to entertain a better way?

Reply:
Added name